src/app/workout/plans/form.tsx

```
@@ -0,0 +1,102 @@
+                />
+                <FormHelperText id="plan-name-input-text" sx={{fontSize: '13px!important'}}>Enter a name for the Workout Plan.</FormHelperText>
+            </FormControl>
+
+            <FormControl sx={{ m: 1, minWidth: 120 }}>
+                <InputLabel id="simple-select-label" sx={{fontSize: '25px!important'}}>Goals</InputLabel>
+                <Select
+                    labelId="simple-select-label"
+                    id="simple-select"
+                    value={values?.goalId}
+                    label="Age"
+                    onChange={(event: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)